app/backend/src/couchers/servicers/api.py

```diff
@@ -619,10 +669,9 @@ def user_model_to_pb(db_user, session, context):
         countries_lived=db_user.countries_lived.split("|") if db_user.countries_lived else [],
         additional_information=db_user.additional_information,
         friends=get_friends_status(session, context.user_id, db_user.id),
-        mutual_friends=[
-            api_pb2.MutualFriend(user_id=mutual_friend.id, username=mutual_friend.username, name=mutual_friend.name)
-            for mutual_friend in db_user.mutual_friends(context.user_id)
-        ],
+        mutual_friends=API.ListMutualFriends(
+            self=None, context=context, request=api_pb2.ListMutualFriendsReq(user_id=db_user.id)
```

Comment:
I'm sure this can be done better, but I'm not sure how. I feel like using self=None is a codesmell

Reply:
Good question, I think it is ugly too. Please make a helper function on the top level, that don't take the "self" parameter.